.github/workflows/optimize-e2e-tests-sm.yml

```diff
@@ -92,6 +103,9 @@ jobs:
           project_name: e2e
           additional_flags: "--profile self-managed"
 
+      - name: Build backend
+        run: mvn clean install -DskipTests -Dskip.docker -pl optimize/backend -am -P skipFrontendBuild
```

Comment:
we can add `-T2C` to make it run in parallel. I used this approach in the unit tests